src/main/java/org/icgc/argo/workflow_management/service/NextflowService.java

```diff
@@ -195,43 +194,38 @@ private CmdKubeRun createCmd(@NonNull Launcher launcher, @NonNull WESRunParams p
     // Dynamic engine properties/config
     val workflowEngineOptions = params.getWorkflowEngineParams();
 
-    if (nonNull(workflowEngineOptions)) {
-
-      // Resume workflow by name/id
-      if (nonNull(workflowEngineOptions.getResume())) {
-        cmdParams.put("resume", workflowEngineOptions.getResume());
-      }
-      // Use revision if provided in workflow_engine_options
-      if (nonNull(workflowEngineOptions.getRevision())) {
-        cmdParams.put("revision", workflowEngineOptions.getRevision());
-      }
-
-      // Use launchDir, projectDir and/or workDir if provided in workflow_engine_options
-      if (nonNull(workflowEngineOptions.getLaunchDir())
-          || nonNull(workflowEngineOptions.getProjectDir())
-          || nonNull(workflowEngineOptions.getWorkDir())) {
-        val config =
-            createNextflowConfigFile(
-                runName,
-                workflowEngineOptions.getLaunchDir(),
-                workflowEngineOptions.getProjectDir(),
-                workflowEngineOptions.getWorkDir());
-        cmdParams.put("runConfig", List.of(config));
-      }
+    // Write config file for run (minimum contents == runAsUser)
+    // Use launchDir, projectDir and/or workDir if provided in workflow_engine_options
```

Comment:
I would change this to
```
// Write config file for run using required and optional arguments
```
Comments can become stale overtime,  especially after renames or removing/adding params. If anyone wants to know whats required and whats optional, they can inspect `createNextflowConfigFile`